Added The Hunter COtW project

PredatorCZ opened the pull request:
Added the Hunter Call of the Wild project with filenames for arc/tab extaction.
Please state any issues to me.

PredatorCZ committed:
Added The Hunter COtW project

gibbed commented:
I don't have this game. Is this game based on the same engine tech as JC3? Same file format besides arc/tab?

PredatorCZ committed:
Added DLC support

Added "dlc_win64" scan folder to RebuilFileLists.
Added filenames for dlc content.

PredatorCZ commented:
Yes, this game uses identical engine.

PredatorCZ committed:
Added Generation Zero project

PredatorCZ committed:
Added 7% of files for Generation Zero project

PredatorCZ committed:
Projects update

Added 31% of files into Hunter COtW project.
Added 3% of files into Generation Zero project.

PredatorCZ committed:
Update filelists for JC3. 68130/186455 (36%)
